docs/book/src/tasks/installation.md

```diff
@@ -243,38 +77,117 @@ stringData:
   username: "<my vCenter username>"
   password: "<my vCenter password>"
 EOF
-
-$ kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-vsphere" asset:"infrastructure-components.yaml" version:"0.5.x"}}
 ```
 
-Check the [vSphere provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/releases) for an up-to-date components file.
-
 For more information about prerequisites, credentials management, or permissions for vSphere, visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/blob/master/docs/getting_started.md).
 
 {{#/tab }}
 {{#tab OpenStack}}
 
-Check the [OpenStack provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/releases) for an up-to-date components file.
+Please visit the [OpenStack getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/getting-started.md).
 
-For more detailed information, e.g. about prerequisites visit the [getting started guide](https://github.com/kubernetes-sigs/cluster-api-provider-openstack/blob/master/docs/getting-started.md).
+{{#/tab }}
+{{#tab Metal3}}
 
-```bash
-kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-openstack" asset:"infrastructure-components.yaml" version:"0.2.x"}}
+Please visit the [Metal3 getting started guide](https://github.com/metal3-io/cluster-api-provider-metal3/blob/master/docs/getting-started.md).
+
+{{#/tab }}
+{{#/tabs }}
+
+## Install clusterctl
+The clusterctl CLI tool handles the lifecycle of a Cluster API management cluster.
+
+{{#tabs name:"install-clusterctl" tabs:"linux,macOS"}}
+{{#tab linux}}
+
+#### Install clusterctl binary with curl on linux
+Download the latest release; for example, to download version v0.3.0 on linux, type:
+```
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.3.0/clusterctl-linux-amd64 -o clusterctl
```

Comment:
```suggestion
curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api" asset:"clusterctl-linux-amd64" version:"0.3.x"}} -o clusterctl
```

This should do it

Reply:
@vincepri this returns `curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/v0.0.0/clusterctl-linux-amd64 -o clusterctl` :-(

Reply:
That's fine, once we publish v0.3.0 it should work properly